src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
```

Comment:
You've declared font-family: "Roboto", sans-serif; in .card and .navigation. This can be declared once in a higher-level selector like body and it will be inherited by all the child elements. This makes your code DRY (Don't Repeat Yourself), easier to maintain, and improve load times (by reducing the size of the CSS file).